README.md

```diff
@@ -116,6 +116,47 @@ PAF type definitions are organized into classes. Classes are currently not used
 
 When type name references are required in the PAF file, two forms are applicable. Either a class name can be given, which will include all types of that class or a ClassName.TypeName form can be used to reference a single type only.
 
+## Analysis Modes
+An analysis is one or a series of operations which uses QTM to generate output data. It can use QTM measurements in the data tree to do calculations and export files. Exactly what the analysis does and which files it uses is defined in the PAF YAML file. See [Overview of analyses](#overview-of-analyses) for more information about analyses.
```

Comment:
Friendly suggestion to change to something along the lines of: **"An analysis is one or more operations which..."**

Reply:
Fixed